courses/forms.py

```
@@ -1,12 +1,12 @@
 from django import forms
 from django.utils.text import slugify
 
-from .models import Courses, CourseTeacher
+from .models import Course, CourseTeacher
```

Ruff (0.8.2)

`.models.CourseTeacher` imported but unused

Remove unused import: `.models.CourseTeacher`

(F401)